xslt/mmd-to-geonorge.xslt

```diff
@@ -95,7 +117,7 @@
 	                <!--title (M) multiplicity [1]-->
                         <xsl:apply-templates select="mmd:title[@xml:lang = 'en']" />
                         <xsl:apply-templates select="mmd:dataset_citation/mmd:publication_date" />
-                        <xsl:apply-templates select="mmd:last_metadata_update" />
+                        <!--xsl:apply-templates select="mmd:last_metadata_update" /-->
```

Comment:
If you are to remove this line, just delete it. Else it can confuse in the future wondering why is this commented out.